test/WebSites/MvcTagHelpersWebSite/Controllers/MvcTagHelper_HomeController.cs

```diff
@@ -70,5 +72,52 @@ public IActionResult Index()
         {
             return View();
         }
+
+        public IActionResult ProductEdit()
+        {
+            ViewBag.Items = _products;
```

Comment:
stick to sensible patterns: don't use same `ViewBag` entry for multiple types of information.  in this case, would expect to see `View(_products)` at line 80.